submissions/Sveti-Stefan/hooli-style-popup/styles.css

```diff
@@ -0,0 +1,196 @@
+body {
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+}
+.container {
+  display: block;
+  width: 100%;
+  height: 100px;
+  background-color: #0c86df;
+}
+.container_header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 100px;
+  width: 100%;
+}
+.logo_header {
+  display: flex;
+  align-content: center;
+  height: 50px;
+}
+.menu_header_nav {
+  display: flex;
+  height: 100px;
+  align-items: center;
+}
+.menu_header_list {
+  display: flex;
+  list-style-type: none;
+  justify-content: flex-end;
+  margin: 0px;
+}
+.menu_header_list_item {
+  padding: 40px;
+  color: white;
+  text-decoration: none;
+}
+
+.menu_header_list_item a {
+  color: white;
+  text-decoration: none;
+  font-size: 24px;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+}
+
+.popup_checkbox {
+  appearance: none;
+}
+
+.popup_checkbox:checked ~ .popup {
+  display: block;
+}
+
+.popup {
+  display: none;
+  position: absolute;
+  background-color: #fff;
+  border-radius: 10px;
+  border: #0c86df solid 2px;
+  width: 336px;
+  height: 280px;
+  left: -205px;
+  top: 116px;
+  overflow-y: auto;
+}
+
+.popup_icon {
+  cursor: pointer;
+  height: 42px;
+}
+
+.popup_item_list {
+  display: grid;
+  grid-template: repeat(2, 1fr) / repeat(3, 80px);
+  grid-auto-flow: row;
+  grid-gap: 14px;
+  justify-items: center;
+  align-items: center;
+  padding: 20px 20px 0px;
+}
+
+.popup_item_list a {
+  color: #0c86df;
+  width: 100px;
+  display: inline-block;
+  text-align: center;
+}
+
+.popup_item_list li {
+  align-content: space-around;
+  width: 80px;
+  height: 80px;
+  padding-bottom: 10px;
+  padding: 10px;
+}
+
+.popup_item_list img {
+  display: inline-block;
+  width: 30px;
+  height: 30px;
+  align-items: center;
+}
+
+.popup_item_list span {
+  align-items: center;
+  font-size: 18px;
+  display: block;
+}
+
+.popup_checkbox_more:checked ~ .popup_more {
+  display: block;
+}
+
+.popup_checkbox_more:checked ~ .more_label {
+  display: none;
+}
+
+.popup_checkbox_more {
+  appearance: none;
+  display: none;
+}
+
+.more_label {
+  color: #0c86df;
+  font-size: 24px;

```

Comment:
Prefer font-scale units for sizing and spacing wherever it's possible (to be fair, it's possible almost everywhere). Such a scale enables creating dynamic sizing when needed when pixel-scale doesn't.

Reply:
Fixed